.github/workflows/ci.yml

```diff
@@ -94,4 +110,4 @@ jobs:
           SILX_TEST_LOW_MEM: "False"
           SILX_OPENCL: ${{ matrix.with_opencl && 'True' || 'False' }}
         run: |
-          python -c "import silx.test, sys; sys.exit(silx.test.run_tests(verbosity=1, args=['--qt-binding=${{ matrix.QT_API }}']));"
+          python run_tests.py --installed --qt-binding=${{ matrix.QT_API }}
```

Comment:
Why is run_tests.py need here?

Reply:
Needed not really, but it is actually much simpler ...